### ISSUE TITLE
Duplicate entries in the terminal drop-down

### ISSUE THREAD

**Issue by kieferrm** [microsoft/vscode-js-debug#786]
Using codespaces PPE with insiders:

![image](https://user-images.githubusercontent.com/4674940/95111982-2906d580-06f5-11eb-860f-aa0d8f9be173.png)

**Comment by connor4312** [comment]
@kieferrm do you have both `js-debug` and `js-debug-nightly` enabled? We recommend having exactly one of these enabled at once.

**Comment by kieferrm** [comment]
Good catch. This happened because I got nightly through setting sync and the js-debug through the codespace by default. It seems that the disable state was not synced correctly.
